code/modules/atmospherics/gasmixtures/space_mixture.dm

```diff
@@ -18,7 +18,7 @@
 	return 0 //we're immutable.
 
 /datum/gas_mixture/space/heat_capacity()
-	. = 700000
```

Comment:
PR title is faulty. I did the calculations and this only makes it 100x less potent.